src/autotype/AutoTypeMatchModel.h

```diff
@@ -49,6 +49,7 @@ class AutoTypeMatchModel : public QAbstractTableModel
     QVariant headerData(int section, Qt::Orientation orientation, int role = Qt::DisplayRole) const override;
 
     void setMatchList(const QList<AutoTypeMatch>& matches);
+    QList<AutoTypeMatch>* matchList();
```

Comment:
I don't like this at all, we should not be returning a pointer to an internal data member. This indicates an antipattern. Instead you should create a function (or use an existing) that searches the match list for the provided match and returns the index.

Reply:
Added a new one since we are fine with a match that matches only the entry but not the sequence.